transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -137,6 +140,17 @@ public B channelFactory(io.netty.channel.ChannelFactory<? extends C> channelFact
         return channelFactory((ChannelFactory<C>) channelFactory);
     }
 
+    /**
+     * Disable calling out to any {@link ChannelInitializerExtension}s for the channels created by this bootstrap.
+     * <p>
+     * Channel initializer extensions are loaded and installed by default, but can be selectively disabled per bootstrap
+     * by calling this method.
+     */
+    public B disableChannelInitializerExtensions() {
```

Comment:
PR descriptions lists very good examples. In my experience, when someone needs to apply extensions, they know better what they are doing compare to those who created a library on top of netty. As someone who wants to apply security or metrics handler, I will be unhappy if one of the libraries in my classpath explicitly disabled extensions. It breaks they whole purpose of an application owner to be able to apply an extension consistently for all Netty traffic.

I would propose to not add this flag. If there will be a legit use-case for that with a good example, we can always add it later.